WooCommerce/Classes/ViewRelated/Coupons/CouponDetails/CouponDetailsViewModel.swift

```diff
@@ -56,6 +60,10 @@ final class CouponDetailsViewModel: ObservableObject {
         }
     }
 
+    var shouldShowErrorLoadingAmount: Bool {
```

Comment:
just curious, does this trigger view reload when either `hasErrorLoadingAmount` or `hasWCAnalyticsDisabled` changes, which are both @Published vars while this one is a computed var?

Reply:
Yes, since the `@Published` variables trigger reloading the view (through `objectWillChange`), the view will need to access the computed variable again and therefore get the latest value.